services/web-server/src/loaders/auth.js

```diff
@@ -15,22 +15,33 @@ module.exports = (clients, isAuthed, rootUrl, monitor, strategies, req, cfg) =>
           throw unauthorizedError;
```

Comment:
I left the above code unwrapped since it seemed like the throw unauthorizedError was intentional and probably be handled by the upstream caller. Also if req.user is undefined for one query it will be undefined for the others, so they would all fail anyways.

Reply:
That makes a lot of sense. A comment that says what you just commented here would probably help for posterity :)